Style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-image: url(Header-pic.png);
+  background-size: cover;
```

Comment:
As the previous reviewer suggested, kindly adjust the design of the headline section to match the figma template.

Modify the background image to as to be centred on the page. Also adjust margins as shown in the illustration below:

Your design:
![image](https://user-images.githubusercontent.com/20554583/212042875-deb9133b-239e-41e4-b048-74b6c6e338c5.png)



Figma design:
![image](https://user-images.githubusercontent.com/20554583/212043324-e1603f09-05fd-4295-b966-492fa0e3cb87.png)